src/config/relay-chains.tsx

```diff
@@ -82,12 +83,14 @@ case POLKADOT: {
   WRAPPED_TOKEN = InterBtc;
   COLLATERAL_TOKEN = Polkadot as Currency<CollateralUnit>;
   // TODO: Add GovernanceUnit type to lib following upgrade i.e. Currency<GovernanceUnit>
-  GOVERNANCE_TOKEN = Interlay as Currency<CurrencyUnit>;
+  GOVERNANCE_TOKEN = Interlay as Currency<GovernanceUnit>;
   WRAPPED_TOKEN_SYMBOL = 'interBTC';
   COLLATERAL_TOKEN_SYMBOL = 'DOT';
   GOVERNANCE_TOKEN_SYMBOL = 'INTR';
   RELAY_CHAIN_NAME = 'polkadot';
-  PRICES_URL = `https://api.coingecko.com/api/v3/simple/price?ids=bitcoin,${RELAY_CHAIN_NAME}&vs_currencies=usd`;
+  BRIDGE_PARACHAIN_NAME = 'interlay';
+  // eslint-disable-next-line max-len
+  PRICES_URL = `https://api.coingecko.com/api/v3/simple/price?ids=bitcoin,${RELAY_CHAIN_NAME},${BRIDGE_PARACHAIN_NAME}&vs_currencies=usd`;
```

Comment:
minor: we've made a couple of recent changes (`TERMS_AND_CONDITIONS_LINK`, `CROWDLOAN_LINK`) which have URLs imported from `utils/links.ts` in case where they differ for  each parachain, rather than constructed using template literals as is being done here. Not a critical issue at all, but would be good to handle this consistently.

Reply:
I think the prices URL should live somewhere outside this file in the future. Since we'll import a couple of prices that might be independent from the chain, we can think about it in future PRs.